dex/ws/wslink.go

```diff
@@ -13,6 +13,7 @@ import (
 	"sync/atomic"
 	"time"
 
+	"decred.org/dcrdex/dex"
```

Comment:
This makes me wish `Error` was in a separate package.

Reply:
I don't follow.  Do you foresee circular dependencies?

Reply:
No. It's fine. I'm just a little concerned that `dex` is becoming a dumping ground for random stuff like `Error` and `Runner` mixed in with the bulk of the package which is dex-specific stuff like locktime and network constants, `Asset`, and `MarketInfo`.

Reply:
The actual `dex` package is kind of a `common` in my mind, where the "sub"packages under it are the dex-specific stuff.  We could have "decred.org/dcrdex/errors" or broader "decred.org/dcrdex/base", but I'm more inclined to keep the repo root tight.

Reply:
Oh, not applicable to this PR anymore, but I'd be fine with moving Error out of `dex` and into such a top level package.

Reply:
I was actually thinking **decred.org/dcrdex/dex/error**, but yeah, not important.